src/gui/mod.rs

```diff
@@ -80,6 +86,13 @@ impl Default for Ajour {
             config: Config::default(),
             expanded_addon: None,
             is_showing_settings: false,
+            shared_client: Arc::new(
+                HttpClient::builder()
+                    .redirect_policy(RedirectPolicy::Follow)
+                    .max_connections_per_host(6)
+                    .build()
+                    .unwrap(),
```

Comment:
Program can panic here, not sure how you want to handle this inside `<Ajour as Application>::new`

Reply:
Good question. For now I think its okay to continue and tackle this task while doing improvements later.